benchmarks/benchAllocTasks.cpp

```diff
@@ -0,0 +1,66 @@
+#include <mc_rbdyn/RobotLoader.h>
+#include <mc_rbdyn/Robots.h>
+#include <mc_solver/QPSolver.h>
+#include <mc_tasks/MetaTaskLoader.h>
+#include <mc_tasks/SurfaceTransformTask.h>
+#include <mc_tasks/lipm_stabilizer/StabilizerTask.h>
+
+#include "benchmark/benchmark.h"
+
+static void BM_AllocSurfaceTransformTask(benchmark::State & state)
+{
+  auto rm = mc_rbdyn::RobotLoader::get_robot_module("JVRC1");
+  mc_rbdyn::Robots robots;
+  robots.load(*rm);
+  for(auto _ : state)
+  {
+    auto task = std::make_shared<mc_tasks::SurfaceTransformTask>("LeftFoot", robots, 0);
+  }
+}
+
+BENCHMARK(BM_AllocSurfaceTransformTask);
+
+static void BM_SurfaceTransformTaskFromConfig(benchmark::State & state)
+{
+  mc_rtc::Configuration config("config.yaml");
```

Comment:
I think you want `@CMAKE_CURRENT_SOURCE_DIR@/config.yaml` and configure the .cpp (same with `config_lipm.yaml`